build/common.sh

```diff
@@ -94,7 +94,7 @@ readonly KUBE_CONTAINER_RSYNC_PORT=8730
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
   local arch=$1
-  local debian_base_version=v2.0.0
+  local debian_base_version=v2.1.0
   local debian_iptables_version=v12.0.1
```

Comment:
this debian_iptables_version has changed in the rest of the PR, should be changed here too?

Reply:
@aojea -- Not yet. See here: https://github.com/kubernetes/kubernetes/pull/90697/files#diff-0ea054266fe6ea2dc38e6079e8adb98bL116

From https://github.com/kubernetes/kubernetes/pull/90665:
>     * Add base images to dependencies.yaml
>       Dependencies for each image are broken up into two sections to allow the dependency check to pass:
>       
>       * `image:tag`: versions that should be bumped in order to build the image
>       * `image:tag: dependents`: dependents that need to be bumped _after_ the image has been built and promoted

So we're bumping the tag just in the Makefile to trigger the image build for debian-iptables in this PR.
After this merges and the image is promoted, I'll come back with another PR to bring the dependents up to the newly-promotion version.

Reply:
thanks for the clarification, I'm just trying to follow this new process, seems a nice improvement

Reply:
Appreciate the feedback, @aojea! 💕
I'll be tracking everything here: https://github.com/kubernetes/kubernetes/issues/90698

...and documenting everything once some of the last quirks are worked out. :)